README.md

```diff
@@ -7,6 +7,9 @@
   <a href="https://github.com/tobealive/wthrr-the-weathercrab/actions/workflows/ci.yml?query=branch%3Amain" target="_blank">
     <img alt="crates.io" src="https://img.shields.io/github/actions/workflow/status/tobealive/wthrr-the-weathercrab/ci.yml?branch=main&style=flat-square" />
   </a>
+  <a href="https://github.com/tobealive/wthrr-the-weathercrab" target="_blank">
+    <img alt="github.com" src="https://img.shields.io/github/last-commit/tobealive/wthrr-the-weathercrab?style=flat-square" />
```

Comment:
This badge is mostly related to GitHub only, so I assumed entering "github.com" as `alt` value would be more appropriate than "crates.io".